gpustack/worker/serve_manager.py

```diff
@@ -245,7 +245,7 @@ def _start_serve_process(self, mi: ModelInstance):  # noqa: C901
             process.start()
             self._serving_model_instances[mi.id] = process
             for port in mi.ports:
-                self._serving_model_instance_ports.add(port)
+                self._serving_model_instance_ports[port] = mi.id
```

Comment:
Can we use id as the key? It's the pattern for other dicts

Reply:
Do we have the case that, one model instance is using multiple ports? Assuming it has this case, is it ok to change to Dict[<id>, list[<port>]]?

Reply:
Yes, we have that in distributed MindIE. Allocated ports are recoreded in instance.ports.

Reply:
Updated